components/01-atoms/links/link.twig

```diff
@@ -0,0 +1,23 @@
+{#
+ # Available variables:
+ # - link_content - the content of the link (typically text)
+ # - link_url - the url this link should poing to
+ # - link_attributes - array of attribute,value pairs
+ #
+ # Available blocks:
+ # - link_content - used to replace the content of the link
+ #     Example: to insert the image component
+ #}
+
+{% set link_base_class = link_base_class|default('link') %}
+
```

Comment:
Yes, there's actually a note in the PR details that says the icons are still undecided (whether we're going to use Font Awesome, or something else) so that'll come in a later PR

Reply:
ah! missed the note. All good here then :+1: